src/Robin/Ntlm/Message/NtlmV1AuthenticateMessageEncoder.php

```diff
@@ -229,6 +241,10 @@ public function encode(
     /**
      * Calculates the LM response.
      *
+     * TODO: Remove this method as it's no longer necessary.
```

Comment:
Since the last release contained a BC break would it be fine to do the same here?